extensions/amp-autocomplete/0.1/amp-autocomplete.css

```diff
@@ -13,3 +13,32 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
+.i-amphtml-autocomplete {   
+    display: inline-flex;
+    flex-direction: column;
+    align-items: stretch;
+ }
+
+.i-amphtml-autocomplete-input {
+    padding: 4px;
+ }
+
+ .i-amphtml-autocomplete-results {
+    margin-top: 4px;
```

Comment:
Huh yeah I also haven't seen this before. I can't think of any reason it would be bad, but I'd also have erred on the side of `position: absolute`. Two additional things:

a) Carlos' concern about 4px of jumping can be solved by taking the margin-top off `-autocomplete-results` (which is the thing with height: 0) and putting it on `.i-amphtml-autocomplete-item:first-child`

b) One thing to consider here is where the results div is parented. Parenting it here is *much* cleaner (and may be necessary for a11y), but has the disadvantage that it can be cut off by any ancestors with `overflow: hidden`. I wouldn't *expect* AMP pages to have complicated layouts that would make this a problem, but it's also nearly impossible to work around if you have a case that needs `overflow: hidden`.